keyboards/feels/feels65/readme.md

```diff
@@ -0,0 +1,13 @@
+# Feels65
+
+65% Keyboards
+
+* Keyboard Maintainer: Swiftrax
+* Hardware Supported: Feels65 PCB
+* Hardware Availability: https://github.com/swiftrax
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader for this board, too?  Eg, hardware reset button, bootmagic, etc.